notebooks/225-distilbert-sequence-classification/225-distilbert-sequence-classification.ipynb

```diff
@@ -0,0 +1,293 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "abc41ac0",
+   "metadata": {},
+   "source": [
+    "# Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fe80a355",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from transformers import AutoTokenizer\n",
+    "import openvino.runtime as ov\n",
+    "import warnings\n",
+    "from pathlib import Path\n",
+    "import numpy as np\n",
+    "import time"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4dde5b43",
+   "metadata": {},
+   "source": [
+    "# Transformers Serialization"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1742fd4b",
+   "metadata": {},
+   "source": [
+    "[ONNX](https://onnx.ai/) is a representation format for deep learning models that allows AI developers to easily transfer models between different frameworks. It is hugely popular among deep learning tools, like PyTorch, Caffe2, Apache MXNet, Microsoft Cognitive Toolkit, and many others. The transformers model from Huggingface needs to be converted into ONNX format to be further converted to OpenVINO IR format. This process to exporting a model to ONNX format is known as [Serialization](https://huggingface.co/docs/transformers/serialization)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5db803ea",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!python -m transformers.onnx -h"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4794f066",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "checkpoint = \"distilbert-base-uncased-finetuned-sst-2-english\"\n",
+    "serialize_command = f\"python -m transformers.onnx \\\n",
```

Comment:
@arunimaaaa according to comment in https://github.com/openvinotoolkit/openvino_notebooks/pull/715#discussion_r1050975102
we can not use this lib for converting models to onnx, because it introduces new dependencies which are not in our IP plan. Is it possible to convert model using another way?

Reply:
Made the changes